compiler+runtime/test/jank/reader-macro/cstring/pass-small-cstring.jank

```diff
@@ -0,0 +1,3 @@
+#cpp "hello"
```

Comment:
We need a test to ensure the escaping is handled properly. i.e. `#cpp "\"foo\"\n\\bar"` needs to come out right on the other end.